helpers/compare.go

```diff
@@ -0,0 +1,13 @@
+package helpers
+
+import "bytes"
+
+// AreInterfacesEqual checks if two interface are equal
+func AreInterfacesEqual(first, second interface{}) bool {
+	switch first.(type) {
+	case []byte:
+		return bytes.Equal(first.([]byte), second.([]byte))
+	default:
+		return true //first == second
```

Comment:
Hmm, I'm not sure about this. I think we should unconditionally cast to bytes

Reply:
@seunlanlege Replaced explicit compare with byte array compare

Reply:
@seunlanlege the problem is comparing two interfaces that may contain byte array data
https://go101.org/article/type-system-overview.html#types-not-support-comparison